package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@fontsource/dancing-script": "^5.0.15",
+    "@libresign/vue-pdf-editor": "^1.0.0",
```

Comment:
Will we use this package or the package `xcc-pdf-editor` ?

About the other changes, will we really need to add all of these packages? Looking the .vue files I didn't found reference to all of the follow packages that was added to package.json

Reply:
the repository of lib code.

https://github.com/LibreSign/vue-pdf-editor/